packages/goblint/goblint.2.0.0/opam

```diff
@@ -0,0 +1,89 @@
+opam-version: "2.0"
+synopsis: "Static analysis framework for C"
+maintainer: [
+  "Simmo Saan <simmo.saan@gmail.com>"
+  "Michael Schwarz <michael.schwarz93@gmail.com>"
+]
+authors: [
+  "Simmo Saan"
+  "Michael Schwarz"
+  "Julian Erhard"
+  "Sarah Tilscher"
+  "Ralf Vogler"
+  "Kalmer Apinis"
+  "Vesal Vojdani"
+]
+license: "MIT"
+homepage: "https://goblint.in.tum.de"
+doc: "https://goblint.readthedocs.io/en/latest/"
+bug-reports: "https://github.com/goblint/analyzer/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "2.9.1"}
+  "goblint-cil" {>= "2.0.0"}
+  "batteries" {>= "3.4.0"}
+  "zarith" {>= "1.8"}
+  "yojson" {>= "2.0.0"}
+  "qcheck-core"
+  "ppx_deriving"
+  "ppx_deriving_hash"
+  "ppx_deriving_yojson" {>= "3.7.0"}
+  "ppx_blob" {>= "0.6.0"}
+  "ounit2" {with-test}
+  "qcheck-ounit" {with-test}
+  "odoc" {with-doc}
+  "fpath"
+  "dune-site"
+  "json-data-encoding"
+  "jsonrpc" {>= "1.12"}
+  "sha" {>= "1.12"}
+  "cpu"
+  "arg-complete"
+  "yaml"
+  "uuidm"
+  "conf-gmp" {>= "3"}
+  "conf-ruby" {with-test}
+  "benchmark" {with-test}
+  "conf-gcc"
+]
+depopts: ["apron" "z3"]
+conflicts: [
+  "result" {< "1.5"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test & os-distribution != "ol"}
```

Comment:
Now that the oraclelinux fix has been merged (thanks) I suppose we cane remove that
```suggestion
    "@runtest" {with-test}
```